test/helpers/config_file.rb

```diff
@@ -0,0 +1,16 @@
+class TestConfigFileBase < Minitest::Test
```

Comment:
This seemed like the most appropriate place for this file, but let me know if there's another convention I should be following